electron/package_action.sh

```diff
@@ -17,7 +17,15 @@
 yarn do electron/build
 
 cp package.json build/windows/
+
+# Environment variables.
 scripts/environment_json.sh -p windows > build/windows/www/environment.json
+# TODO: Share code with environment_json.sh.
+mkdir -p build/windows/build
+cat > build/windows/build/env.nsh << EOF
+!define RELEASE "$(node -r fs -p 'JSON.parse(fs.readFileSync("package.json")).version;')"
+!define SENTRY_DSN "https://sentry.io/api/159503/store/?sentry_version=7&sentry_key=319145c481df41458bb6e84c1a99c9ff"
```

Comment:
Worth adding a comment that this is a debug Sentry DSN for outline-clients-dev?

Reply:
Done.